relay/polkadot/src/lib.rs

```diff
@@ -667,7 +667,12 @@ parameter_types! {
 		"DOT_SLASH_DEFER_DURATION"
 	);
 	pub const RewardCurve: &'static PiecewiseLinear<'static> = &REWARD_CURVE;
-	pub const MaxNominatorRewardedPerValidator: u32 = 512;
+	// TODO:(PR#137) - check MaxExposurePageSize/MaxNominators 512?
+	pub const MaxExposurePageSize: u32 = 512;
+	// Note: this is not really correct as Max Nominators is (MaxExposurePageSize * page_count) but
+	// this is an unbounded number. We just set it to a reasonably high value, 1 full page
+	// of nominators.
+	pub const MaxNominators: u32 = 512;
```

Comment:
@gpestana maybe you can check this?

Reply:
This configuration needs to be reviewed by Jakob from SR labs as well.
https://github.com/paritytech-secops/srlabs_findings/issues/327

Reply:
plus somebody should check  `MaxExposurePageSize` also